docs/img/omnisci/omnisci_query_flow.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The diagram looks overloaded, I wonder which parts of it and how I should simplify? Suggestions?

Reply:
You can try to re-arrane to make it narrower but taller given current page layout.

Crude suggestion (beware, mad Paint skillz!):
![image](https://user-images.githubusercontent.com/45396231/144873165-0db47b3a-555b-43e6-9099-de561fa9d722.png)

Reply:
[done](https://modin--3753.org.readthedocs.build/en/3753/flow/modin/experimental/core/execution/native/implementations/omnisci_on_native/index.html#data-transformation-flow)